lib/dom-event-delegate.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @fileoverview An object that encapsulates event delegation wireup for a
+ * DOM element.
+ * @author Box
+ */
+
+Box.DOMEventDelegate = (function() {
+
+	'use strict';
+
+	// Supported events for modules. Only events that bubble properly can be used in T3.
+	var eventTypes = ['click', 'mouseover', 'mouseout', 'mousedown', 'mouseup',
```

Comment:
I can change it. This was the way it was in application.js, so I mindlessly copied. Probably should be all uppercase.